docs/coding-practices.md

```diff
@@ -0,0 +1,123 @@
+# Coding Practices
+
+Author : Cédric Gautier
+Date : 2024-01-01
```

Comment:
Useless, it's a tracked file, you have it from the commit, plus `git log` will provide all the versionning.

If you keep it here, there's a chance that someone will forget updating the date when they will update the doc. Better leave the reader check the date from git.

Reply:
Same for the author.